### DISCUSSION TITLE
How to perform a factory reset to the entire orcaslicer program (profiles and printers)

### DISCUSSION THREAD

**Discussion by lucas0jeda** [OrcaSlicer/OrcaSlicer#5781] [Q&A]
I have a serious problem with orcaslicer, something went wrong at the time of printing. I did a flow and pressure calibration but I saved the changes generated by the TEST, now I don't know what was changed and now my printer prints wrong. I want to discard all the changes to return to a factory default state in the printer profiles.

**Comment by vgdh** [comment]
just delete it's data folder
%appdata%/orcaslicer/

**Reply by lucas0jeda** [reply]
In MacOs??

**Reply by vgdh** [reply]
sorry I don't know how to do it in macOs

**Reply by SoftFever** [reply]
On Mac, it's in Application Support folder: ~/Library/Application Support

**Comment by srvrguy** [comment]
For macOS, there are a few locations. Depending on your usage, not all of these may exist.

**Key Locations to Remove**
These locations will contain saved settings or other data used by OrcaSlicer that should be removed to clear all settings.
- Directory ~/Library/Application Support/OrcaSlicer/
- File ~/Library/Preferences/com.softfever3d.orca-slicer.plist


**Non-critical**
These locations are less critical as they consist of caches and Mac-internal stuff relating to the application itself, but useful if you want to clean out _all_ data stored or used by OrcaSlicer.
- Directory ~/Library/Caches/com.softfever3d.orca-slicer/
- Directory ~/Library/Saved Application State/com.softfever3d.orca-slicer.savedState/
- Directory ~/Library/WebKit/com.softfever3d.orca-slicer/
